src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1406,4 +1406,16 @@ This version used in: {2}</value>
   <data name="Convert_to_linq_call_form" xml:space="preserve">
     <value>Convert to LINQ (call form)</value>
   </data>
+  <data name="Remove_unused_member" xml:space="preserve">
+    <value>Remove unused member</value>
+  </data>
+  <data name="Avoid_Unused_Members_Message" xml:space="preserve">
+    <value>Type '{0}' has an unused private member '{1}' which can be removed.</value>
+  </data>
+  <data name="Avoid_Unused_Members_Title" xml:space="preserve">
+    <value>Avoid unused private members.</value>
```

Comment:
1. IDE resource strings should match resource values.  This should be "Avoid_unused_private_members".
2. do we generally have periods in these strings?  I dont' think we do...